src/allmydata/_auto_deps.py

```diff
@@ -123,6 +123,8 @@
     ('enum34',           'enum'),
     ('pycparser',        'pycparser'),
     ('PyYAML',           'yaml'),
+    ('magic-wormhole',   'wormhole'),
+    ('setuptools',       'setuptools')
```

Comment:
hm, why are these additions necessary? Most of this PR is about removing things (yay!), are these needed to make up for something being removed elsewhere?

(my long-term hope is to get rid of `_auto_deps.py` and move everything into `setup.py` or `setup.cfg`)

Reply:
`setuptools` was indeed being added to this list (in the next codeblock, I believe) but only for non-frozen builds; if `setuptools` is in `install_requires` but not present here, frozen builds fail too.

The `magic-wormhole` addition should have been added previously with the `tahoe invite` stuff. Without it, frozen builds fail at run-time (while adding it will correct show magic-wormhole's version in `tahoe --version` which, I think, is the desired behavior).